packages/teams-js/src/public/runtime.ts

```diff
@@ -18,8 +18,9 @@ export interface IRuntime {
       readonly update?: {};
     };
     readonly location?: {};
-    readonly log?: {};
+    readonly logs?: {};
     readonly mail?: {};
+    readonly media?: {};
```

Comment:
Merge conflict resolution gone wrong? I think that the change to remove `media` from the runtime happened since you last pulled, so there should not be references to `media` in the runtime.ts.